periodic_job.go

```diff
@@ -87,6 +87,18 @@ func (s *periodicIntervalSchedule) Next(t time.Time) time.Time {
 	return t.Add(s.interval)
 }
 
+type neverSchedule struct{}
+
+func (s *neverSchedule) Next(t time.Time) time.Time {
+	// Return the maximum future time so that the schedule never runs.
+	return time.Unix(1<<63-1, 0)
+}
+
+// ScheduleNever returns a PeriodicSchedule that never runs.
+func ScheduleNever() PeriodicSchedule {
+	return &neverSchedule{}
+}
```

Comment:
Naming is a little tough here. `neverSchedule` is consistent with the internal name `periodicIntervalSchedule` above, but for the external API name I thought `ScheduleNever()` makes more sense. If we're going to provide one-off schedule options, they should probably be grouped with a common prefix `ScheduleXX` for discoverability.

None are consistent with `PeriodicInterval`'s name though.

Let me know what you think.

Reply:
Hrmm, I'm a little mixed on this one. Given that `PeriodicSchedule` is already an external naming scheme, it may be that `NeverSchedule` fits better into that, don't you think? "Never schedule" also flows off the tongue a bit easier.

Reply:
Fair point, renamed to `NeverSchedule` 👍